lib/registries/addon/drafts/draft/metadata/template.hbs

```diff
@@ -103,19 +106,28 @@
                     for={{subjectsFieldId}}
                     id='subjects'
                 >
-                    {{t 'registries.registration_metadata.subjects'}}
+                    <p local-class='DisplayText'>
+                        {{t 'registries.registration_metadata.subjects'}}
+                        <span local-class='Required'>*</span>
+                    </p>
                 </label>
                 <Subjects::Manager
                     @model={{this.draftManager.draftRegistration}}
                     @provider={{this.draftManager.draftRegistration.provider}}
                     @doesAutosave={{true}}
+                    @metadataChangeset={{this.draftManager.metadataChangeset}}
                     as |subjectsManager|
                 >
                     <Subjects::Widget
                         id={{subjectsFieldId}}
                         @subjectsManager={{subjectsManager}}
                     />
                 </Subjects::Manager>
+                <ValidationErrors
+                    @changeset={{this.draftManager.metadataChangeset}}
+                    @key='subjects'
+                    @isValidating={{this.draftManager.metadataChangeset.isValidating}}
```

Comment:
I think `isValidating` is method on the changeset not a computed property. So you could add a CP maybe on the draftManager that calls the method?

Reply:
Will add a computed on the metadata route controller.